detectable/src/test/java/com/synopsys/integration/detectable/detectables/maven/unit/MavenCodeLocationPackagerTest.java

```diff
@@ -176,6 +176,17 @@ public void testIsGav() {
         assertTrue(mavenCodeLocationPackager.isGav("group:artifact:type:classifier:version:scope:garbage"));
     }
 
+    @Test
+    public void testIsEclipsePackage() {
+        MavenCodeLocationPackager mavenCodeLocationPackager = new MavenCodeLocationPackager(new ExternalIdFactory());
+
+        String line = "[INFO] +- p2.eclipse-plugin:org.eclipse.core.runtime:jar:3.17.100.v20200203-0917:system";
+        line = mavenCodeLocationPackager.trimLogLevel(line);
+        String cleanedLine = mavenCodeLocationPackager.calculateCurrentLevelAndCleanLine(line);
+        Dependency dependency = mavenCodeLocationPackager.textToDependency(cleanedLine);
+        assertEquals("org.eclipse.core.runtime/3.17.100.v20200203-0917", dependency.getExternalId().createExternalId());
```

Comment:
If possible it would be nice to check namespace (forge?) here as well.

Reply:
Very good point, done!